src/CoreBundle/Resources/config/services.xml

```diff
@@ -950,8 +950,6 @@
 
         <service id="chameleon_system_core.module.navigation_tree_single_select_wysiwyg" class="ChameleonSystem\CoreBundle\Bridge\Chameleon\Module\NavigationTreeSingleSelectWysiwyg\NavigationTreeSingleSelectWysiwyg" shared="false">
```

Comment:
You can further simplify (and correct) this by giving a "parent":

parent="chameleon_system_core.module.navigation_tree_single_select"
(behind class)

All the dependencies except the request_stack can and must be removed then here.

Reply:
OK, done.